src/Application.php

```diff
@@ -1098,11 +1098,11 @@ public function container(string $name, $value=null): mixed
 
     } elseif (\is_callable($value)) {
       # Callable
-      $this->getContainer()->addShared($name,$value);
+      $this->getContainer()->share($name,$value);
```

Comment:
This dep has been updated from 3 to version 5, the new name method name is addShared, so this is breaking the tests